fastapitableau/rstudio_connect.py

```diff
@@ -29,7 +29,7 @@ def warning_message() -> Optional[str]:  # noqa: C901
     message_list: List[str] = []
 
     connect_server = environ.get("CONNECT_SERVER")
-    # TODO DEBUG: Print CONNECT_SERVER variable
+    # Debug message: Print CONNECT_SERVER variable
```

Comment:
What is with these?

Reply:
I wanted to remove the "TODO" comments, but leave in a marker for myself about where to post log messages from when I implement debug logging.

Reply:
Removed